cluster/gce/upgrade.sh

```diff
@@ -258,6 +258,7 @@ function prepare-node-upgrade() {
   KUBE_PROXY_TOKEN=$(get-env-val "${node_env}" "KUBE_PROXY_TOKEN")
   export KUBE_PROXY_TOKEN
   NODE_PROBLEM_DETECTOR_TOKEN=$(get-env-val "${node_env}" "NODE_PROBLEM_DETECTOR_TOKEN")
+  export NODE_PROBLEM_DETECTOR_TOKEN
```

Comment:
So I'm not sure about the effects of not exporting it but all the other vars were getting exported so I'm also exporting this var

Reply:
Looks like they are consumed by util.sh according to https://github.com/kubernetes/kubernetes/pull/88576/files#r405561884